tests/execute/ignore-duration/test.sh

```
@@ -0,0 +1,45 @@
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+        rlRun "export TMT_WORKDIR_ROOT=$tmp"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest "No envvar used"
+        rlRun -s "tmt run -vv plan -n /no-option" "2"
+        rlAssertGrep 'errr /demo/test (timeout)' $rlRun_LOG '-F'
```

Shellcheck (0.10.0)

[warning] rlRun_LOG is referenced but not assigned.

(SC2154)